locales-id-ID.xml

```diff
@@ -69,13 +69,13 @@
     <term name="page-range-delimiter">–</term>
 
     <!-- ORDINALS -->
-    <term name="ordinal"></term>
-    <term name="ordinal-01"></term>
-    <term name="ordinal-02"></term>
-    <term name="ordinal-03"></term>
-    <term name="ordinal-11"></term>
-    <term name="ordinal-12"></term>
-    <term name="ordinal-13"></term>
+    <term name="ordinal">bilangan bertingkat</term>
```

Comment:
I think you meay have misunderstood this? the "ordinal" term refers to what gets added to a number to turn it into an ordinal. So in English for numbers >3 that's "th" -- 4th, 5th, 6th, 7th etc. Unfortunately, we can't do prefixes, if that's how this works in Indonesian?